### ISSUE TITLE
Simplify menu

### ISSUE THREAD

**Issue by octavioamu** [gitcoinco/web#5485]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a user I want to have a better and consolidated menu in all the site to help me find the areas of the site in any place.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:
Today you need to click a lot to get into explorer, and is a most needed area, each product you need to navigate in two steps away to find what you need, is not a good UX.

### Description

Create 2 menu levels for each product. 
Gitcoin/company - what, how, etc 
Bounties - Sub bounties
Hackathon - Sub Hackathon
Grants - Sub grants
Kudos - Sub kudos
Quests - Sub quests

Notifications in all pages not only internals

Example:
![image](https://user-images.githubusercontent.com/1358093/68701792-4a535600-0566-11ea-8b1a-1272a9eeb068.png)

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
Hard to navigate, and understand the sitemap 
### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Better experience and discoverability 
You can move to another subarea product from everywhere/discover new products. 
### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

### Data Requirements
[comment]: # (How will we measure the success of this feature? What kind of tracking is needed for this feature (clicks, impressions, flag)?)

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)
![image](https://user-images.githubusercontent.com/1358093/68702921-6657f700-0568-11ea-9a17-7cc6d187ba41.png)

**Comment by octavioamu** [comment]
Design question, should be time to move our menu to vertical menu?
cc @willsputra

**Comment by willsputra** [comment]
Will need to do further research on this, but my concern with putting each product name as the main nav is that new users might not know what Bounties, Grants, Kudos, etc. is about. I agree that having a way to access the Issue Explorer easily will be beneficial for existing users, but I wonder how we can do it while also catering to new users.

I personally love a 'use case'-based approach ('what Gitcoin products help me achieve specific goals?' instead of 'here's a list of Gitcoin products'), like Shopify or Plaid here:

![Screenshot 2019-11-26 23 49 00](https://user-images.githubusercontent.com/19514207/69649125-84b10d00-10a7-11ea-9fd1-6947c7a63aae.png)

![Screenshot 2019-11-26 23 49 07](https://user-images.githubusercontent.com/19514207/69649150-8d094800-10a7-11ea-8af0-89bf5af203ca.png)

**Comment by octavioamu** [comment]
we can discuss in some call, I don't think shopify or plaid are good to compare since their have a product paid for use, is the normal landing. When we actually offer a set of diff products to help to an end goal but not necessary to use all. I don't see the actual menu is helping people to really get to places where they get interested to start  working on a bounty for example. And we are making it harder for the ones to get into the explorer. 
We can setup a call and analyse the flows data if you prefer.

**Comment by octavioamu** [comment]
Call with @willsputra :
Looking at out analytics we saw there is a problem with our navigation architecture. Will is going to create an architecture map of the current nav to demostrate all the step a user need to do to get to diff places and then propose a new one.

The solution we found is create a **mega menu** for products ex of companies with subproducts using mega menu:
https://azure.microsoft.com/
![image](https://user-images.githubusercontent.com/1358093/69826660-e3cf6880-11f2-11ea-85a4-f73430c7b9f3.png)

https://aws.amazon.com/
![image](https://user-images.githubusercontent.com/1358093/69826674-f649a200-11f2-11ea-9834-3431e13d4245.png)

With this, users will be able to jump into sub product areas from everywhere without need to go around different levels of the sites.
Also better for mobile users. 

Problems this solution will solve:
Hard to get into the explorer (and some other pages) , people use to memorize the url instead of using the nav some of them never get into the explorer. 
High drop-off on bounty/founder
No notifications in main page
Dashboard link hidden.
Names / menus consolidations: "bounty" link is sometimes Explorer and others bounties/funder making really hard to understand where the link goes.
